tests/tracer/test_buffer.py

```diff
@@ -1,82 +0,0 @@
-import pytest
-
-from ddtrace.internal.buffer import BufferFull
-from ddtrace.internal.buffer import BufferItemTooLarge
-from ddtrace.internal.buffer import TraceBuffer
-
-
-def test_buffer_put_get():
-    buf = TraceBuffer(max_size=10, max_item_size=3)
-    buf.put([1, 2, 3])
-    assert len(buf) == 1
-    assert buf.size == 3
-
-    buf.put([3, 2])
-    assert len(buf) == 2
-    assert buf.size == 5
-
-    items = buf.get()
-    assert len(buf) == 0
-    assert buf.size == 0
-    assert items == [[1, 2, 3], [3, 2]]
-
-    buf.put("123")
-    buf.put("45")
-    buf.put("67")
-    buf.put("89")
-    buf.put("0")
-    assert buf.size == 10
-    assert len(buf) == 5
-    assert buf.get() == ["123", "45", "67", "89", "0"]
-    assert buf.size == 0
-
-
-def test_buffer_size_limit():
-    buf = TraceBuffer(max_size=10, max_item_size=3)
-    buf.put([3])
-    for i in range(9):
-        buf.put([0])
-
-    with pytest.raises(BufferFull):
```

Comment:
I don't see corresponding tests for these ones that have been deleted. I think we might be covered by existing tests in `test_integration` and `test_writer` but just wanted to confirm that we're not losing any coverage here.

Reply:
Good point! I'll adapt some of these tests to the new API.